lib/migrations/util/timestamp.js

```diff
@@ -1,16 +1,14 @@
-// Get a date object in the correct format, without requiring a full out library
-// like "moment.js".
 function yyyymmddhhmmss() {
-  const d = new Date();
+	const now = new Date(Date.now());
```

Comment:
Why use `new Date(Date.now());` together instead of either of these two?

Reply:
Date.now() returns milliseconds from 1970 to now, to my understanding there is no integer prototype for .getUTCSeconds, so you have to convert it to a new Date() object.